pest_derive/Cargo.toml

```diff
@@ -17,7 +17,8 @@ proc-macro = true
 [dependencies]
 quote = "^0.3"
 syn = "^0.10"
-pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
+# pest = { git = "https://github.com/pest-parser/pest", branch = "procedural" }
```

Comment:
@Keats Oh, true. I think it's fine to merge on procedural if fails on Travis and then see if it passes there.